### PR TITLE
Updated rate limiting section

### PR DESCRIPTION
**Pull request by isaacdurham-okta**
## Description:
- **What's changed?** Updated ASA rate limiting section
- **Is this PR related to a Monolith release?** 

### Resolves:

* [OKTA-446295](https://oktainc.atlassian.net/browse/OKTA-446295)

### CODE REVIEW

**Review by phoebeyu-okta — APPROVED** [review]

**Review by SajanAlexander-okta — APPROVED** [review]
LGTM

File: packages/@okta/vuepress-site/docs/reference/api/asa/introduction/index.md [hunk 1]
```diff
@@ -105,18 +105,14 @@ An example of a paginated response body that contains a `list` field:
...
-  permitted during the current rate limit period
-* `X-RateLimit-Reset` - contains the unix timestamp for the beginning of the
-  next rate limit period
+Rate limits control access to Advanced Server Access APIs by measuring the rate at which users send requests during a rate limit period. After exceeding this limit, users must wait for requests to replenish before submitting again.
+
+Every API response includes headers related to rate limiting:
+
+* `X-RateLimit-Limit` - Describes the total number of operations permitted per rate limit period.
+* `X-RateLimit-Remaining` - Describes the total number of remaining operations permitted during the current rate limit period.
+* `X-RateLimit-Reset` - Contains a unix timestamp indicating when the number of rate limit requests will fully replenish.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/asa/introduction/index.md:114]:
```suggestion
* `X-RateLimit-Reset` - Contains a unix timestamp indicating when the number of rate limit requests fully replenishes.
```

File: packages/@okta/vuepress-site/docs/reference/api/asa/introduction/index.md [hunk 2]
```diff
@@ -105,18 +105,14 @@ An example of a paginated response body that contains a `list` field:
...
+* `X-RateLimit-Retry-At` - Sent after exceeding the rate limit and contains a unix timestamp indicating when the number of rate limit requests will partially replenish. If multiple requests share the same timestamp, sending them at the same time may cause additional rate limiting.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/asa/introduction/index.md:115]:
```suggestion
* `X-RateLimit-Retry-At` - Sent after exceeding the rate limit and contains a unix timestamp indicating when the number of rate limit requests partially replenishes. If multiple requests share the same timestamp, sending them at the same time may cause additional rate limiting.
```

**Review by brentschaus-okta — APPROVED** [review]
LGTM

A few suggestions pertaining to future tense.

**Pull request merged**